Formula/g/gnuplot.rb

```diff
@@ -97,339 +94,27 @@ def install
-   if test "$QTVER" = 5; then
-       { printf "%s\n" "$as_me:${as_lineno-$LINENO}: result:   Qt terminal: yes (qt5)" >&5
- printf "%s\n" "  Qt terminal: yes (qt5)" >&6; }
-+  fi
-+  if test "$QTVER" = 6; then
-+      { printf "%s\n" "$as_me:${as_lineno-$LINENO}: result:   Qt                   : yes (qt6)" >&5
-+printf "%s\n" "  Qt                   : yes (qt6)" >&6; }
-   fi
- else
-   { printf "%s\n" "$as_me:${as_lineno-$LINENO}: result:   Qt terminal: no (use --with-qt or --with-qt=qt4 to enable" >&5
-diff -pur gnuplot-5.4.8/configure.ac gnuplot-5.4.8-new/configure.ac
---- gnuplot-5.4.8/configure.ac	2023-06-02 01:13:08
-+++ gnuplot-5.4.8-new/configure.ac	2023-07-31 14:26:04
-@@ -990,29 +990,48 @@ if test "${enable_qt}" = yes ; then
-       enable_qt_ok=no
-   fi
- 
--  dnl First check for Qt5
--  if test "x${with_qt}" = "xqt5"; then
--    try_qt4=no
--  else
-+  dnl Qt4 only by special request
-+  if test "x${with_qt}" = "xqt4"; then
-     try_qt4=yes
-+  else
-+    try_qt4=no
-   fi
-+
-   if test "x${with_qt}" != "xqt4"; then
--    PKG_CHECK_MODULES_NOFAIL(QT, [Qt5Core Qt5Gui Qt5Network Qt5Svg Qt5PrintSupport])
-+    AC_MSG_CHECKING([Checking for Qt6 support libraries])
-+    PKG_CHECK_MODULES_NOFAIL(QT, [Qt6Core Qt6Gui Qt6Network Qt6Svg Qt6PrintSupport Qt6Core5Compat])
-     if test $pkg_failed = no; then
--      try_qt4=no
--      QT5LOC=`$PKG_CONFIG --variable=host_bins Qt5Core`
--      if test "x${QT5LOC}" != "x"; then
--        UIC=${QT5LOC}/uic
--        MOC=${QT5LOC}/moc
--        RCC=${QT5LOC}/rcc
--        LRELEASE=${QT5LOC}/lrelease
-+      QT6LOC=`$PKG_CONFIG --variable=host_bins Qt6Core`
-+      if test "x${QT6LOC}" != "x"; then
-+        UIC=${QT6LOC}/uic
-+        MOC=${QT6LOC}/moc
-+        RCC=${QT6LOC}/rcc
-+        LRELEASE=${QT6LOC}/lrelease
+diff --git a/configure b/configure
+index c918ea8..019dcc0 100755
+--- a/configure
++++ b/configure
+@@ -16393,12 +16393,17 @@ fi
+           UIC=${QT6LOC}/uic
+           MOC=${QT6LOC}/moc
+           RCC=${QT6LOC}/rcc
+-      else
++      elif test "x${UIC}" = "x"; then
+           UIC=uic-qt6
+           MOC=moc-qt6
+           RCC=rcc-qt6
        fi
-       CXXFLAGS="$CXXFLAGS -fPIC"
-+      AC_MSG_RESULT([The Qt terminal will use Qt6.])
-+      QTVER="6"
-+    else
-+      AC_MSG_CHECKING([Checking for Qt5 support libraries])
-+      PKG_CHECK_MODULES_NOFAIL(QT, [Qt5Core Qt5Gui Qt5Network Qt5Svg Qt5PrintSupport])
-+      if test $pkg_failed = no; then
-+        QT5LOC=`$PKG_CONFIG --variable=host_bins Qt5Core`
-+        if test "x${QT5LOC}" != "x"; then
-+          UIC=${QT5LOC}/uic
-+          MOC=${QT5LOC}/moc
-+          RCC=${QT5LOC}/rcc
-+          LRELEASE=${QT5LOC}/lrelease
-+        fi
-+        CXXFLAGS="$CXXFLAGS -fPIC"
-+        AC_MSG_RESULT([The Qt terminal will use Qt5.])
-+        QTVER="5"
+-      LRELEASE=lrelease-qt6
++      QT6BIN=`$PKG_CONFIG --variable=bindir Qt6Core`
++      if test "x${QT6BIN}" != "x"; then
++          LRELEASE=${QT6LOC}/lrelease
```

Comment:
```suggestion
+          LRELEASE=${QT6BIN}/lrelease
```
This should fix the linux build

Reply:
BTW, the upstream patch is actually correct.

Reply:
I must've changed it when testing something. Thanks for pointing that out.